documentscanner/src/main/res/values/strings_document_scanner.xml

```diff
@@ -0,0 +1,26 @@
+<resources>
+    <string name="app_name">MEGA Document Scanner</string>
```

Comment:
Please, also add a context for the app name string. I'm sure that translators wil require it.
For example: "Name of the MEGA Document Scanner. Keep uppercase."

```suggestion
    <string name="app_name" context="Name of the MEGA Document Scanner. Keep uppercase.">MEGA Document Scanner</string>
```